gltfTutorial/README.md

```diff
@@ -0,0 +1,18 @@
+#glTF Tutorial
+
+This tutorial gives an introduction to [glTF](https://www.khronos.org/gltf), the GL transmission format. It summarizes the most important features and application cases of glTF, and describes the structure of the files that are related to glTF. It explains how glTF files may be read, processed and used to display 3D graphics efficiently.
+
+Some basic knowledge about [JSON](http://json.org/), the JavaScript Object Notation, and about [OpenGL](https://www.khronos.org/opengl/) is required. Where appropriate, the related concepts of OpenGL will be explained quickly, usually with examples in [WebGL](https://www.khronos.org/webgl/).

```

Comment:
This is a bit confusing worded as is.  Perhaps say something like "knowledge of a graphics API...examples will be in WebGL."